velox/exec/RowContainer.h

```diff
@@ -260,11 +260,20 @@ class RowContainer {
     *ptr = std::min<uint64_t>(size, std::numeric_limits<uint32_t>::max());
   }
 
-  // Initialize row. 'reuse' specifies whether the 'row' is reused or
-  // not. If it is reused, it will free memory associated with the row
-  // elsewhere (such as in HashStringAllocator).
+  /// Initialize row. 'reuse' specifies whether the 'row' is reused or not. If
+  /// it is reused, it will free memory associated with the row elsewhere (such
+  /// as in HashStringAllocator).
+  /// Note: Fields of the row is not zero-initialized. If the row contains
+  /// variable-width fields and the caller does not intend to write to those
```

Comment:
nit:  the caller must populate these fields by calling 'store' or initialize them to zero by calling 'initializeFields'.